arango/batch.py

```diff
@@ -214,6 +247,32 @@ def graph(self, name):
         return Graph(self, name)
 
 
+if six.PY2:
+    class Python2Lock(object):
+        # https://stackoverflow.com/questions/35149889/lock-with-timeout-in-python2-7
+        def __init__(self):
+            self._underlying = Queue.Queue(maxsize=1)
```

Comment:
I think `_underlying` should be changed to  `_queue` for just straightforward clarity.

Reply:
Sure